deployment/workshop/README.md

```diff
@@ -192,10 +192,13 @@ We have each student install Keycloak as part of a security lab but you need to
 
 * As admin, Goto the OpenShift webconsole and Operators > OperatorHub
 * Filter by keyword "Keycloak" and you should see a community operator show up - Click it.
-* The details will slide in from the right, Click Install (we are using version 8.0.2 today, higher should be OK too)
-* Install for every user project 
-  * eek, that's a lot! I know there's a CLI for this - *TODO replace GUI with CLI instructions*
-  * we plan to automate all this so don't worry it'll be easier soon
+* The details will slide in from the right. Note the version and see that it matches the version in the
+* file keycloak/keycloak-operator-install.sh
+*
+* Install for every user project using the following script
+```
+   keycloak/keycloak-operator-install.sh
```

Comment:
The keycloak install script is not executable by default.  Let's remove the script and use `oc create` directly here.

Reply:
or I could change it to
```
  sh keycloak/keycloak-operator-install.sh

Reply:
You could just change the default rwx mode on the file.  Still, I like the idea of being consistent so we don't have the keycloak part using a script but the rest not.

Reply:
I'll accept the PR and add the `sh`.  Thanks!